spec/controllers/admin/entry_templates_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+module Admin
+  describe EntryTemplatesController do
+    render_views
+
+    describe '#create' do
+      it 'still displays form after submit attempt if data is invalid' do
+        admin = create(:user, :admin)
+        sign_in(admin, scope: :user)
+
+        post(:create,
+             params: {
+               account_id: admin.accounts.first.id,
+               entry_template: {
+                 theme_name: 'unregistered'
+               }
+             })
+
+        expect(response.body).to have_selector(
+          '.input.error#entry_template_theme_name_input'
+        )
+      end
+    end
+
+    describe '#edit' do
+      it 'still displays form after submit attempt if data is invalid' do
+        entry_template = create(:entry_template)
+        sign_in(create(:user, :admin), scope: :user)
+
+        patch(:update,
+              params: {
+                account_id: entry_template.account.id,
+                id: entry_template.id,
+                entry_template: {
+                  theme_name: 'unregistered'
+                }
+              })
+
+        expect(response.body).to have_selector(
+          '.input.error#entry_template_theme_name_input'
+        )
+      end
+    end
+  end
```

Comment:
I'd look a the account admin spec for more things to test here. Code related to updating nested entry templates can in turn be removed there.

Reply:
Indeed. After working on the entry template admin for a while today, I noticed that some account controller specs should be ported to entry template controller before doing more in that direction